### PR TITLE
Add sample for service chaining

### PR DESCRIPTION
**Pull request by ThishaniLucas**
Sample for https://github.com/wso2/ballerina-integrator/issues/9

### CODE REVIEW

File: examples/integration-tutorials/exposing-several-services-as-a-single-service/README.md [hunk 1]
```diff
@@ -1 +1,30 @@
-# exposing-several-services-as-a-single-service
+# Service Chaining
+
+In this example, we're going to integrate many service calls and expose a single service.
```
Comment (Asitha) [inline examples/integration-tutorials/exposing-several-services-as-a-single-service/README.md]:
```suggestion
In this example, we're going to integrate many service calls and expose as a single service. This is commonly referred to as Service Chaining, where several services are integrated based on some business logic and exposed as a single, aggregated service.
```

File: examples/integration-tutorials/exposing-several-services-as-a-single-service/README.md [hunk 2]
```diff
@@ -1 +1,30 @@
...
+
+## Backend  
+
+We will use the same hospital service as the backend for this example. The hospital service exposes multiple services i.e making an appointment reservation, viewing appointment details, viewing the doctors' details, etc. We will combine two of such services, making an appointment reservation and settling the payment for the reservation, and expose them as a single service in ballerina.  
+
+## Developement
+
+In this example, we will expose a service in ballerina named 'makeReservation'. A client will call this service with the request payload.  This service will first call the reservation service of the backend and obtain a response. Using that response, it will call the payment settlement service of the backend and obtain a response. Finally it will send the response back to the client.
+
```
Comment (Asitha) [inline examples/integration-tutorials/exposing-several-services-as-a-single-service/README.md]: Let's walk through the code segments and explain what it does as well

File: examples/integration-tutorials/exposing-several-services-as-a-single-service/reserve_appointment_service.bal [hunk 3]
```diff
@@ -0,0 +1,83 @@
+import ballerina/http;
```
Comment (Asitha) [inline examples/integration-tutorials/exposing-several-services-as-a-single-service/reserve_appointment_service.bal]: Lets add WSO2 License header

File: examples/integration-tutorials/exposing-several-services-as-a-single-service/reserve_appointment_service.bal [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+import ballerina/log;
+
+http:Client hospitalEP = new("http://localhost:9090");
```
Comment (Asitha) [inline examples/integration-tutorials/exposing-several-services-as-a-single-service/reserve_appointment_service.bal]: We need to add variable and function comments

File: examples/integration-tutorials/exposing-several-services-as-a-single-service/test/reserve_appointment_service_test.bal [hunk 5]
```diff
@@ -0,0 +1,36 @@
+import ballerina/test;
```
Comment (Asitha) [inline examples/integration-tutorials/exposing-several-services-as-a-single-service/test/reserve_appointment_service_test.bal]: License header?

File: examples/integration-tutorials/exposing-several-services-as-a-single-service/README.md (hunk 1 above)
Comment (ThishaniLucas) [reply examples/integration-tutorials/exposing-several-services-as-a-single-service/README.md]: done

File: examples/integration-tutorials/exposing-several-services-as-a-single-service/README.md (hunk 2 above)
Comment (ThishaniLucas) [reply examples/integration-tutorials/exposing-several-services-as-a-single-service/README.md]: done

File: examples/integration-tutorials/exposing-several-services-as-a-single-service/reserve_appointment_service.bal (hunk 3 above)
Comment (ThishaniLucas) [reply examples/integration-tutorials/exposing-several-services-as-a-single-service/reserve_appointment_service.bal]: done

File: examples/integration-tutorials/exposing-several-services-as-a-single-service/reserve_appointment_service.bal (hunk 4 above)
Comment (ThishaniLucas) [reply examples/integration-tutorials/exposing-several-services-as-a-single-service/reserve_appointment_service.bal]: done

File: examples/integration-tutorials/exposing-several-services-as-a-single-service/test/reserve_appointment_service_test.bal (hunk 5 above)
Comment (ThishaniLucas) [reply examples/integration-tutorials/exposing-several-services-as-a-single-service/test/reserve_appointment_service_test.bal]: done

**Pull request merged**